server/controllers/tickets.js

```diff
@@ -22,6 +22,33 @@ module.exports.getPanelTickets = (req, res) => {
     });
 };
 
+/** TODO: WRITE TEST FOR THIS FUNCTION **/
+module.exports.getPanelTicketsByUser = (req, res) => {
+  if (helper.checkUndefined(req.query.panel_id)) {
+    res.status(400).send('one of parameters from client is undefined');
+    return;
+  }
+  
+  var panelId = req.query.panel_id;
+  dbhelper.getTicketsByPanel(panelId)
```

Comment:
Without converting to integer with parseInt(panelId), there's a chance that dbhelper will confuse the argument as a string and fail. I've had those failures before which is why I do parseInt on anything that is supposed to be an integer when passing into a dbhelper function.